cutil/cutil.go

```diff
@@ -796,3 +797,30 @@ func GetDockerEndpoint() string {
 	dockerEP := "unix://" + listenerSocket
 	return dockerEP
 }
+
+// extrace the version in certificate file. The version should be in the first line in this format -----x.x.x-----
+func GetCertificateVersion(certFilePath string) string {
+	var firstLine string
+	var version string
+
+	glog.Info("try another way to read")
+	if cert, err := os.Open(certFilePath); err != nil {
+		glog.Info(fmt.Sprintf("unable to open Certificate file %v, error %v", certFilePath, err))
+	} else if certBytes, err := ioutil.ReadAll(cert); err != nil {
+		glog.Info(fmt.Sprintf("unable to read ESS SSL Certificate file %v, error %v", certFilePath, err))
+	} else {
+		lines := strings.Split(string(certBytes), "\n")
+		if len(lines) > 0 {
+			firstLine = lines[0]
+		}
+		if firstLine != "" {
+			parts := strings.Split(firstLine, "-----")
+			if len(parts) == 3 {
+				version = parts[1]
```

Comment:
What will be returned if the cert file does not contain the version?

Reply:
@linggao If cert file doesn't have version, the version returned will be an empty string. 
The `exchangesync/node.go` will not get certificate version. Then the local db and exchange node will show empty string from cert version

Reply:
@linggao I got you. Yes, you are right. Will add check for that. I think I added that check in `agent-install.sh`, need to check if extract `version != BEGIN CERTIFICATE`

Reply:
for old cert file that starts with "-----BEGIN CERTIFICATE-----", the function will return "BEGIN CERTIFICATE" as the version. Is it right?

Reply:
@linggao yes, will fix it